CHANGELOG.md

```diff
@@ -5,9 +5,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [0.11.0-beta.3] - Unreleased
+### Added
+- Additional overlapping endpoint tests.
+
 ### Changed
 - Add support for aggregated server owned interfaces.
 
+### Fixed
+- Known issue with overlapping endpoints #2.
```

Comment:
"Invalid overlapping endpoints, now they are always refused."

Reply:
Correctly handle parametric endpoints regardless of the ordering, so that overlapping endpoints are always refused. (See #2)